src/content/docs/en/guides/authentication.mdx

```diff
@@ -184,7 +184,9 @@ Configure your database table to store user data and your preferred authenticati
 ```ts title="src/pages/api/auth/[...all].ts"
 import { auth } from "../../../lib/auth"; // import your Better Auth instance
 import type { APIRoute } from "astro";
- 
+
+export const prerender = false; // Prevents static prerendering of this dynamic route
```

Comment:
```suggestion
export const prerender = false; // Not needed in 'server' mode
```
This is the line we use for these code snippets. We don't have to describe what the line does as much as we need to make it clear *when* it should be added, because another alternative is to have your entire app rendered on-demand by default!